### ISSUE TITLE
jctools-parent 1.2 artifact doesn't exist which prevents jctools-core 1.2 from being dependent on.

### ISSUE THREAD

**Issue by xedin** [JCTools/JCTools#97]
It looks like parent has been added in 1.2 I've tried both jitpack.io and maven.org and both fail e.g.

```
[ERROR] Failed to execute goal on project windmill-core: Could not resolve dependencies for project com.github.xedin:windmill-core:jar:0.2-SNAPSHOT: Failed to collect dependencies at org.jctools:jctools-core:jar:1.2: Failed to read artifact descriptor for org.jctools:jctools-core:jar:1.2: Failure to find org.jctools:jctools-parent:pom:1.2 in http://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
```

**Comment by guidomedina** [comment]
@nitsanw did you deploy from jctools-core or from parent? I think in one of my commits I put core and experimental as children of jctools-parent, but parent has packaging as pom so it shouldn't be a problem to deploy it.

**Comment by guidomedina** [comment]
I can undo that change, the only problem is that it is ugly from the IDE perspective as you have 3 projects under a parent but no relationship, is there a hack for that?

**Comment by nitsanw** [comment]
I deploy from core, it's the only releasable part.
I don't mind sorting this out for releasing. Your change broke it, but for good reasons.
I don't see the point in releasing the parent, or experimental/benchmarks.
I have re-released 1.2, it's already on bintray, should hit maven central in the randomly near future.
Apologies.

**Comment by guidomedina** [comment]
I can un-link the core pom from parent again, the drawback with it is the double maintenance cost of properties and versions.
